src/tools/roc_conv/main.cpp

```diff
@@ -69,22 +70,26 @@ int main(int argc, char** argv) {
     pipeline::ConverterConfig config;
 
     if (args.frame_size_given) {
-        if (args.frame_size_arg <= 0) {
-            roc_log(LogError, "invalid --frame-size: should be > 0");
+        if (sizeof(args.frame_size_arg) <= 0) {
+            roc_log(LogError, "invalid --frame-size: frame size unspecified");
             return 1;
         }
-        config.internal_frame_size = (size_t)args.frame_size_arg;
+        roc::core::parse_duration(args.frame_size_arg, config.internal_frame_size);
```

Comment:
We should check the return value of parse_duration(). Also, `roc::` is not necessary here, we have `using namespace roc` directive above.

Reply:
done.